frontend/crd/src/app/milestones-page/milestones/milestones.component.html

```diff
@@ -13,7 +13,7 @@
       <mat-expansion-panel>
         <mat-expansion-panel-header>
           <mat-panel-title>
-            <mat-checkbox [checked]="milestone.isComplete" color="primary" (click)="$event.stopPropagation()"></mat-checkbox>
+            <mat-checkbox [disabled]="true" [checked]="milestone.isComplete" color="primary" (click)="$event.stopPropagation()"></mat-checkbox>
```

Comment:
would just having "disabled" work? only the milestone checkbox was enabled, the task and event checkboxes were successfully disabled. is this more robust or something?

Reply:
Just disabled works, but there is a warning about it. I think that's an old style of doing it and Angular prefers this now